src/manageTeam/manageTeamWebview.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  CancellationToken,
+  commands,
+  ExtensionContext,
+  WebviewView,
+  WebviewViewProvider,
+  WebviewViewResolveContext,
+  window,
+} from "vscode";
+import { layout } from "../utils/webviewLayout";
+import { getHubBaseUrl, isNewerThanVersion } from "../utils/binary.utils";
+
+export function registerManageTeamWebviewProvider(context: ExtensionContext) {
+  const provider = new ManageTeamWebviewProvider();
+  void setManageTeamWebviewReady();
+
+  context.subscriptions.push(
+    window.registerWebviewViewProvider("tabnine-team", provider)
+  );
+}
+
+async function setManageTeamWebviewReady() {
+  if (await isNewerThanVersion([4, 0, 55])) {
```

Comment:
use [isCapabilityEnabled](https://github.com/codota/tabnine-vscode/blob/5840dee427f0f369f51a1a249d843bd6a0d70203/src/capabilities/capabilities.ts#L26) instead, it is controlled via the https://github.com/codota/acpl-paid/blob/master/features.toml file